x-pack/plugins/security_solution/public/management/pages/trusted_apps/view/components/effected_policy_select/index.ts

```diff
@@ -0,0 +1,7 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+export * from './effected_policy_select';
```

Comment:
I've seen that in common components (and in some other projects) they just put the whole component into index.tsx. I was doing the same for other components. What are your thoughts on this?

Reply:
For the most part, I prefer file names are are descriptive -vs- placing entire pieces code in `index.**` files. Makes it (IMO) better DX

Reply:
I guess it's more about consistency.